packages/client/app/routes.ts

```diff
@@ -13,10 +13,7 @@ export default [
 		route("me/posts", "routes/users/me/posts.tsx"),
 		route(":id/answers", "routes/users/:id/answers.tsx"),
 	]),
-	...prefix("posts", [
-		route(":id", "routes/posts/:id.tsx"),
-		route(":id/answer", "routes/posts/:id/answer.tsx"),
-	]),
+	route("posts/:id", "routes/posts/:id.tsx"),
```

Comment:
```suggestion
	route("posts/:id", "routes/posts/index.tsx"),
```
正味どっちでもいいんだけど俺だったらindex.tsxとかにするかも。お任せします